torchci/scripts/updateQueueTimes.mjs

```diff
@@ -19,8 +32,10 @@ const response = await client.queryLambdas.executeQueryLambda(
   {}
 );
 
-console.log(response);
-
-await client.documents.addDocuments("metrics", "queue_times_historical", {
-  data: response.results,
-});
+s3client.send(
+  new PutObjectCommand({
+    Bucket: "ossci-raw-job-status",
+    Key: `queue_times_historical/${response.results[0]._event_time}`,
```

Comment:
This work but I would prefer to see UNIX epoch as the key instead as it looks nicer as S3 path, but that's a minor point.